spec/controllers/saml_idp_controller_spec.rb

```diff
@@ -118,7 +118,7 @@
       result = { service_provider: nil, saml_request_valid: false }
       expect(@analytics).to receive(:track_event).with(Analytics::REMOTE_LOGOUT_INITIATED, result)
 
-      delete :remotelogout, params: { SAMLRequest: 'foo' }
+      post :remotelogout, params: { SAMLRequest: 'foo' }
```

Comment:
This actually wasn't necessary to keep the tests passing so I also added request specs to ensure that we're only permitting POST requests.